source/util/convert_to_iso_string.ts

```diff
@@ -36,6 +36,7 @@ export function convertToIsoString(
     // wrap it new Date() to convert it to UTC based ISO string in case it is in another timezone
     try {
       return new Date(data as any).toISOString();
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove the variable instead

Reply:
can't remove variable in catch. but i found an option in eslint config to ignore this case. i'll enable it instead

Reply:
_ is the normal way of removing, but the eslint config change is probably better

Reply:
fixed the eslint rule!